bevy_ios_safearea/src/lib.rs

```diff
@@ -1,7 +1,7 @@
-#![doc = include_str!("../README.md")]
+#![doc = include_str!("../../README.md")]
```

Comment:
this will not work and is working as expected on main

Reply:
Restored- my solution worked on Windows and main one doesn't. But on the other systems solution from main worked just fine. 
Multiplatform is hard. :(